requirements.in

```diff
@@ -16,6 +16,7 @@ click==7.1.2              # via black, mozilla-jetstream
 coverage==5.3             # via mozilla-jetstream, pytest-cov
 deprecated==1.2.10        # via pygithub
 flake8==3.8.4             # via pytest-flake8
+GitPython==3.1.9          # via mozilla-jetstream
```

Comment:
It looks like a setup.py diff is missing :)

It looks like GitPython shells out to git to do most of its work; what do you think about using a library like pygit2 that wraps libgit2? The cross-platform story looks better -- I suppose it doesn't really matter if Docker is our only deployment platform.